mbox2html.py

```diff
@@ -46,7 +46,10 @@ def get_header_text(msg, item, default='utf-8'):
             except:
                 header_section = text
         else:
-            header_section = text.decode(charset or default)
+            try:
+                header_section = text.decode(charset or default)
+            except:
+                header_section = text.decode(default)
```

Comment:
at that point in the control flow one could opt to ignore utf-8 decoding errors in the last except ( https://docs.python.org/3/howto/unicode.html#the-string-type -> `b'\x80abc'.decode("utf-8", "ignore")`)